packages/fluentui/accessibility/src/behaviors/Card/cardsContainerBehavior.ts

```diff
@@ -15,7 +15,7 @@ const cardsContainerBehavior: Accessibility<CardsContainerBehaviorProps> = () =>
   attributes: {},
   focusZone: {
     props: {
-      shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Enter,
+      shouldEnterInnerZone: event => getCode(event) === keyboardKey.Enter,
```

Comment:
```suggestion
      shouldEnterInnerZone: event => getCode(event) === EnterKey,
```

Let's keep it consistent. Is this exported?

Reply:
This is exported, but not things are available and exported as separate variables. So I don't know that is consistent in this case

Reply:
Oh man, ok got it